SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/WorkerCoordinator.cs

```diff
@@ -0,0 +1,296 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using Improbable.Worker;
+
+namespace Improbable.WorkerCoordinator
+{
+    /// <summary>
+    /// Executable that connects and runs a simulated player.
+    /// The class defines the coordinator worker executable that will run simulated players.
+    ///
+    /// ARGUMENTS
+    /// The first 5 arguments are consumed by the coordinator, and used for connecting the coordinator to
```

Comment:
Seems like the `coordinator_start_delay_millis` is not positional argument?

Reply:
Yeah that's confusing, although the argument doesn't look positional it must be passed as the 5th argument. It used to be that there were more named coordinator arguments which could be provided in any order after the receptionist args and before the client args, but since this is the only arg that survived its position is implicitly fixed. It is possible that additional arguments will be added to the coordinator in the future, so I would leave it as is - discussed offline as well